library/datetime/to_time_spec.rb

```diff
@@ -2,5 +2,24 @@
 require 'date'
 
 describe "DateTime#to_time" do
-  it "needs to be reviewed for spec completeness"
+  it "yields a new Time object" do
+    DateTime.now.to_time.should be_kind_of(Time)
+  end
+
+  ruby_version_is "2.4" do
+    it "preserves the same time regardless of local time or zone" do
+      date = DateTime.new(2012, 12, 24, 12, 23, 00, '+03:00')
+      time = date.to_time
```

Comment:
Almost there :)
But I think this `to_time` needs to move in the `with_timezone` so we can check that `to_time` does not depend on the current timezone.

Reply:
Done. :)